src/main/scala/cromwell/engine/db/slick/WorkflowExecutionComponent.scala

```diff
@@ -59,4 +64,34 @@ trait WorkflowExecutionComponent {
       if workflowExecution.status inSet statuses
     } yield workflowExecution
   }
+
+  /**
+   * Query workflow execution using the filter criteria encapsulated by the `WorkflowQueryParameters`.
+   */
+  def queryWorkflowExecutions(queryParameters: WorkflowQueryParameters) = {
+    val Include: Rep[Boolean] = true
```

Comment:
Specifically for constant naming conventions it doesn't meet this portion of the scala style guide's definition: "... and it belongs to a package object or an object"